test/Conversion/ATenToXTen/aten_conv2d_add_lrelu.mlir

```diff
@@ -0,0 +1,31 @@
+//===- aten_conv2d_nobias.mlir ---------------------------------*- MLIR -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2019 Xilinx Inc.
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: aten-opt %s -aten-to-xten | FileCheck %s
+// CHECK: %5 = "xten.conv2d_tensoradd_lrelu"(%4, %0, %none, %1, %2, %3, %int1, %float4.000000e-01, %arg0) : (!torch.vtensor<[1,2,128,128],f32>, !torch.vtensor<[16,2,3,3],f32>, !torch.none, !torch.list<int>, !torch.list<int>, !torch.list<int>, !torch.int, !torch.float, !torch.vtensor<[1,2,128,128],f32>) -> !torch.vtensor<[1,2,128,128],f32>
```

Comment:
Perhaps also check the result is used by the `return`? You can use pattern-matching to give names to parts of the string.